lagotto/map.jinja

```diff
@@ -13,6 +13,8 @@
     'app_port': 9292,
     'app_root': '/var/www/lagotto',
     'application': "lagotto",
+    'bugsnag_hostname': salt.grains.get('fqdn'),
+    'bugsnag_release_stage': environment,
```

Comment:
These key-values are injected into the container via https://github.com/PLOS-Formulas/lagotto-formula/blob/dev/lagotto/init.sls#L35. Along the way, all the keys are uppercase.